spring-boot-project/spring-boot-docs/src/docs/asciidoc/howto/properties-and-configuration.adoc

```diff
@@ -71,7 +71,7 @@ You can automatically expand properties from the Gradle project by configuring t
 
 [source,gradle,indent=0,subs="verbatim"]
 ----
-	processResources {
+	tasks.named('processResources', Copy) {
```

Comment:
I don't think the task's type is needed here.

Reply:
Yes, it would work without the type.

I've added the explicit type declaration to give the IDE a chance to resolve what `expand` refers to, but at least for IntelliJ IDEA this does not seem to be sufficient.

For proper auto-completion in IntelliJ IDEA we'd need this:
```gradle
tasks.named('processResources', Copy) {
    it.expand(project.properties)
}
```

If we don't care about IDE support, this would be enough:
```gradle
tasks.named('processResources') {
    expand(project.properties)
}
```

Reply:
Thanks. Given that it’s unrelated to configuration avoidance, please revert this part of the changes.